packages/server/assets/accessrequests/typedefs/accessrequests.graphql

```diff
@@ -13,22 +19,64 @@ extend type Stream {
   pendingAccessRequests: [StreamAccessRequest!] @hasStreamRole(role: STREAM_OWNER)
 }
 
+extend type Project {
+  """
+  Pending project access requests.
+  """
+  pendingAccessRequests: [ProjectAccessRequest!] @hasStreamRole(role: STREAM_OWNER)
+}
+
 extend type Mutation {
   """
-  Accept or decline a stream access request. Must be a stream owner to invoke this.
+  DEPRECATED: Accept or decline a stream access request. Must be a stream owner to invoke this.
   """
   streamAccessRequestUse(
     requestId: String!
     accept: Boolean!
     role: StreamRole! = STREAM_CONTRIBUTOR
+  ): Boolean!
+    @hasRole(role: "server:user")
+    @hasScope(scope: "users:invite")
+    @deprecated(reason: "Streams are deprecated in favour of projects.")
+
+  """
+  Accept or decline a stream access request. Must be a stream owner to invoke this.
+  """
+  projectAccessRequestUse(
+    requestId: String!
+    accept: Boolean!
+    role: StreamRole! = STREAM_CONTRIBUTOR
   ): Boolean! @hasRole(role: "server:user") @hasScope(scope: "users:invite")
 
   """
-  Request access to a specific stream
+  DEPRECATED: Request access to a specific stream
   """
   streamAccessRequestCreate(streamId: String!): StreamAccessRequest!
     @hasRole(role: "server:user")
     @hasScope(scope: "users:invite")
+    @deprecated(reason: "Streams are deprecated in favour of projects.")
+
+  """
+  Request access to a specific project.
+  """
+  projectAccessRequestCreate(projectId: String!): ProjectAccessRequest!
+    @hasRole(role: "server:user")
```

Comment:
If we're re-doing large parts of the API I'd vote for moving roles & scopes to GraphQL enums (for the new fields), similarly to `StreamRole`